### ISSUE TITLE
Unable to find FirebaseCppApp-6_10_0

### ISSUE THREAD

**Issue by isassaba** [firebase/quickstart-unity#588]
Unity editor version: 2019.2.5f1
Firebase Unity SDK version: 6.10.0
Firebase plugins in use (Auth, Database, etc.): Cloud Messaging
Additional SDKs you are using (Facebook, AdMob, etc.): OneSignal, Mapbox
Platform you are using the Unity editor on (Mac, Windows, or Linux): Mac
Platform you are targeting (iOS, Android, and/or desktop):  Android
Scripting Runtime (Mono, and/or IL2CPP): IL2CPP

### Please describe the issue here:
I add the unitypackage of firebase i make a build first on mono only and it doesn't find the FirebaseCppApp-6_10_0.
After checking for a while on the issues here, i tried:
-Change to IL2CPP and arm64 and armv7.
-Disabling Auto-Resolution on the resolver
-Force resolve
-Deleting Firebase and reimporting.
-Restart and change the name of the project.

Nothing is working...
Also I attach a .gif of something thats is happening when run the resolver that seems to be the problem because it adds the dlls but then delete them.

![ezgif com-video-to-gif](https://user-images.githubusercontent.com/7903152/74865786-78e48800-5330-11ea-97b1-f75b55049dd9.gif)

This is the error on the app when it starts:

02-19 15:48:43.499: E/Unity(12589): Unable to find FirebaseCppApp-6_10_0
02-19 15:48:43.542: E/Unity(12589): DllNotFoundException: Unable to load DLL 'FirebaseCppApp-6_10_0': The specified module could not be found.
02-19 15:48:43.542: E/Unity(12589):   at Firebase.AppUtilPINVOKE+SWIGExceptionHelper.SWIGRegisterExceptionCallbacks_AppUtil (Firebase.AppUtilPINVOKE+SWIGExceptionHelper+ExceptionDelegate applicationDelegate, Firebase.AppUtilPINVOKE+SWIGExceptionHelper+ExceptionDelegate arithmeticDelegate, Firebase.AppUtilPINVOKE+SWIGExceptionHelper+ExceptionDelegate divideByZeroDelegate, Firebase.AppUtilPINVOKE+SWIGExceptionHelper+ExceptionDelegate indexOutOfRangeDelegate, Firebase.AppUtilPINVOKE+SWIGExceptionHelper+ExceptionDelegate invalidCastDelegate, Firebase.AppUtilPINVOKE+SWIGExceptionHelper+ExceptionDelegate invalidOperationDelegate, Firebase.AppUtilPINVOKE+SWIGExceptionHelper+ExceptionDelegate ioDelegate, Firebase.AppUtilPINVOKE+SWIGExceptionHelper+ExceptionDelegate nullReferenceDelegate, Firebase.AppUtilPINVOKE+SWIGExceptionHelper+ExceptionDelegate outOfMemoryDelegate, Firebase.AppUtilPINVOKE+SWIGExceptionHelper+ExceptionDelegate overflowDelegate

**Comment by paulinon** [comment]
Thanks for reporting this issue @isassaba. Could you try updating your JDK and SDK in your Unity Preferences? If that doesn't work, please send a minimal, reproducible example of your project as well as your API compatibility setting.

**Comment by isassaba** [comment]
Hi @paulinon, thanks for the response I updated them yesterday via Android Studio. Today I saw on a thread that changing "plugins" to "Plugins" may do something. I did that, and now when I "Force resolve" it keeps the dlls on the Plugins folder but when I tried to build I get duplicate errors like this:

`CommandInvokationFailure: Gradle build failed. 
/Applications/Unity2019.2.5f1/PlaybackEngines/AndroidPlayer/Tools/OpenJDK/MacOS/bin/java -classpath "/Applications/Unity2019.2.5f1/PlaybackEngines/AndroidPlayer/Tools/gradle/lib/gradle-launcher-5.1.1.jar" org.gradle.launcher.GradleMain "-Dorg.gradle.jvmargs=-Xmx4096m" "assembleRelease"

stderr[

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':checkReleaseDuplicateClasses'.
> 1 exception was raised by workers:
  java.lang.RuntimeException: Duplicate class android.support.v4.app.INotificationSideChannel found in modules classes.jar (:androidx.core.core-1.0.0:) and classes.jar (:support-compat-25.1.0:)
  Duplicate class android.support.v4.app.INotificationSideChannel$Stub found in modules classes.jar (:androidx.core.core-1.0.0:) and classes.jar (:support-compat-25.1.0:)
  Duplicate class android.support.v4.app.INotificationSideChannel$Stub$Proxy found in modules classes.jar (:androidx.core.core-1.0.0:) and classes.jar (:support-compat-25.1.0:)
  Duplicate class android.support.v4.media.MediaBrowserCompat found in modules classes.jar (:androidx.media.media-1.0.0:) and classes.jar (:support-media-compat-25.1.0:)
  Duplicate class android.support.v4.media.MediaBrowserCompat$CallbackHandler found in modules classes.jar (:androidx.media.media-1.0.0:) and classes.jar (:support-media-compat-25.1.0:)
  Duplicate class android.support.v4.media.MediaBrowserCompat$ConnectionCallback found in modules classes.jar (:androidx.media.media-1.0.0:) and classes.jar (:support-media-compat-25.1.0:)
  Duplicate class android.support.v4.media.MediaBrowserCompat$ConnectionCallback$ConnectionCallbackInternal found in modules classes.jar (:androidx.media.media-1.0.0:) and classes.jar (:support-media-compat-25.1.0:)
  Duplicate class android.support.v4.media.MediaBrowserCompat$ConnectionCallback$StubApi21 found in modules classes.jar (:androidx.media.media-1.0.0:) and classes.jar (:support-media-compat-25.1.0:)
  Duplicate class android.support.v4.media.MediaBrowserCompat$ItemCallback found in modules classes.jar (:androidx.media.media-1.0.0:) and classes.jar (:support-media-compat-25.1.0:)
  Duplicate class android.support.v4.media.MediaBrowserCompat$ItemCallback$StubApi23 found in modules classes.jar (:androidx.media.media-1.0.0:) and classes.jar (:support-media-compat-25.1.0:)
  Duplicate class android.support.v4.media.MediaBrowserCompat$ItemReceiver found in modules classes.jar (:androidx.media.media-1.0.0:) and classes.jar (:support-media-compat-25.1.0:)
  Duplicate class android.support.v4.media.MediaBrowserCompat$MediaBrowserImpl found in modules classes.jar (:androidx.media.media-1.0.0:) and classes.jar (:support-media-compat-25.1.0:)
  Duplicate class android.support.v4.media.MediaBrowserCompat$MediaBrowserImplApi21 found in modules classes.jar (:androidx.media.media-1.0.0:) and classes.jar (:support-media-compat-25.1.0:)
  Duplicate class android.support.v4.media.MediaBrowserCompat$MediaBrowserImplApi21$1 found in modules classes.jar (:androidx.media.media-1.0.0:) and classes.jar (:support-media-compat-25.1.0:)
  Duplicate class android.support.v4.media.MediaBrowserCompat$MediaBrowserImplApi21$2 found in modules classes.jar (:androidx.media.media-1.0.0:) and classes.jar (:support-media-compat-25.1.0:)
  Duplicate class android.support.v4.media.MediaBrowserCompat$MediaBrowserImplApi21$3 found in modules classes.jar (:androidx.media.media-1.0.0:) and classes.jar (:support-media-compat-25.1.0:)
  Duplicate class android.support.v4.media.MediaBrowserCompat$MediaBrowserImplApi23 found in modules classes.jar (:androidx.media.media-1.0.0:) and classes.jar (:support-media-compat-25.1.0:)
  Duplicate class android.support.v4.media.MediaBrowserCompat$MediaBrowserImplBase found in modules classes.jar (:androidx.media.media-1.0.0:) and classes.jar (:support-media-compat-25.1.0:)
  Duplicate class android.support.v4.media.MediaBrowserCompat$MediaBrowserImplBase$1 found in modules classes.jar (:androidx.media.media-1.0.0:) and classes.jar (:support-media-compat-25.1.0:)
  Duplicate class android.support.v4.media.MediaBrowserCompat$MediaBrowserImplBase$2 found in modules classes.jar (:androidx.media.media-1.0.0:) and classes.jar (:support-media-compat-25.1.0:)
  Duplicate class android.support.v4.media.MediaBrowserCompat$MediaBrowserImplBase$3 found in modules classes.jar (:androidx.media.media-1.0.0:) and classes.jar (:support-media-compat-25.1.0:)
  Duplicate class android.support.v4.media.MediaBrowserCompat$MediaBrowserImplBase$MediaServiceConnection found in modules classes.jar (:androidx.media.media-1.0.0:) and classes.jar (:support-media-compat-25.1.0:)
  Duplicate class android.support.v4.media.MediaBrowserCompat$MediaBrowserImplBase$MediaServiceConnection$1 found in modules classes.jar (:androidx.media.media-1.0.0:) and classes.jar (:support-media-compat-25.1.0:)
  Duplicate class android.support.v4.media.MediaBrowserCompat$MediaBrowserImplBase$MediaServiceConnection$2 found in modules classes.jar (:androidx.media.media-1.0.0:) and classes.jar (:support-media-compat-25.1.0:)
  Duplicate class android.support.v4.media.MediaBrowserCompat$MediaBrowserServiceCallbackImpl found in modules classes.jar (:androidx.media.media-1.0.0:) and classes.jar (:support-media-compat-25.1.0:)
  Duplicate class android.support.v4.media.MediaBrowserCompat$MediaItem found in modules classes.jar (:androidx.media.media-1.0.0:) and classes.jar (:support-media-compat-25.1.0:)
  Duplicate class android.support.v4.media.MediaBrowserCompat$MediaItem$1 found in modules classes.jar (:androidx.media.media-1.0.0:) and classes.jar (:support-media-compat-25.1.0:)
  Duplicate class android.support.v4.media.MediaBrowserCompat$MediaItem$Flags found in modules classes.jar (:androidx.media.media-1.0.0:) and classes.jar (:support-media-compat-25.1.0:)
  Duplicate class android.support.v4.media.MediaBrowserCompat$ServiceBinderWrapper found in modules classes.jar (:androidx.media.media-1.0.0:) and classes.jar (:support-media-compat-25.1.0:)
  Duplicate class android.support.v4.media.MediaBrowserCompat$Subscription found in modules classes.jar (:androidx.media.media-1.0.0:) and classes.jar (:support-media-compat-25.1.0:)
  Duplicate class android.support.v4.media.MediaBrowserCompat$SubscriptionCallback found in modules classes.jar (:androidx.media.media-1.0.0:) and classes.jar (:support-media-compat-25.1.0:)
  Duplicate class android.support.v4.media.MediaBrowserCompat$SubscriptionCallback$StubApi21 found in modules classes.jar (:androidx.media.media-1.0.0:) and classes.jar (:support-media-compat-25.1.0:)
  Duplicate class android.support.v4.media.MediaBrowserCompatApi21 found in modules classes.jar (:androidx.media.media-1.0.0:) and classes.jar (:support-media-compat-25.1.0:)
  Duplicate class android.support.v4.media.MediaBrowserCompatApi21$ConnectionCallback found in modules classes.jar (:androidx.media.media-1.0.0:) and classes.jar (:support-media-compat-25.1.0:)
  Duplicate class android.support.v4.media.MediaBrowserCompatApi21$ConnectionCallbackProxy found in modules classes.jar (:androidx.media.media-1.0.0:) and classes.jar (:support-media-compat-25.1.0:)
  Duplicate class android.support.v4.media.MediaBrowserCompatApi21$MediaItem found in modules classes.jar (:androidx.media.media-1.0.0:) and classes.jar (:support-media-compat-25.1.0:)
  Duplicate class android.support.v4.media.MediaBrowserCompatApi21$SubscriptionCallback found in modules classes.jar (:androidx.media.media-1.0.0:) and classes.jar (:support-media-compat-25.1.0:)
  Duplicate class android.support.v4.media.MediaBrowserCompatApi21$SubscriptionCallbackProxy found in modules classes.jar (:androidx.media.media-1.0.0:) and classes.jar (:support-media-compat-25.1.0:)
  Duplicate class android.support.v4.media.MediaBrowserCompatApi23 found in modules classes.jar (:androidx.media.media-1.0.0:) and classes.jar (:support-media-compat-25.1.0:)
  Duplicate class android.support.v4.media.MediaBrowserCompatApi23$ItemCallback found in modules classes.jar (:androidx.media.media-1.0.0:) and classes.jar (:support-media-compat-25.1.0:)
  Duplicate class android.support.v4.media.MediaBrowserCompatApi23$ItemCallbackProxy found in modules classes.jar (:androidx.media.media-1.0.0:) and classes.jar (:support-media-compat-25.1.0:)
  Duplicate class android.support.v4.media.MediaDescriptionCompat found in modules classes.jar (:androidx.media.media-1.0.0:) and classes.jar (:support-media-compat-25.1.0:)
  Duplicate class android.support.v4.media.MediaDescriptionCompat$1 found in modules classes.jar (:androidx.media.media-1.0.0:) and classes.jar (:support-media-compat-25.1.0:)
  Duplicate class android.support.v4.media.MediaDescriptionCompat$Builder found in modules classes.jar (:androidx.media.media-1.0.0:) and classes.jar (:support-media-compat-25.1.0:)
  Duplicate class android.support.v4.media.MediaDescriptionCompatApi21 found in modules classes.jar (:androidx.media.media-1.0.0:) and classes.jar (:support-media-compat-25.1.0:)
  Duplicate class android.support.v4.media.MediaDescriptionCompatApi21$Builder found in modules classes.jar (:androidx.media.media-1.0.0:) and classes.jar (:support-media-compat-25.1.0:)
  Duplicate class android.support.v4.media.MediaDescriptionCompatApi23 found in modules classes.jar (:androidx.media.media-1.0.0:) and classes.jar (:support-media-compat-25.1.0:)
  Duplicate class android.support.v4.media.MediaDescriptionCompatApi23$Builder found in modules classes.jar (:androidx.media.media-1.0.0:) and classes.jar (:support-media-compat-25.1.0:)
  Duplicate class android.support.v4.media.MediaMetadataCompat found in modules classes.jar (:androidx.media.media-1.0.0:) and classes.jar (:support-media-compat-25.1.0:)
  Duplicate class android.support.v4.media.MediaMetadataCompat$1 found in modules classes.jar (:androidx.media.media-1.0.0:) and classes.jar (:support-media-compat-25.1.0:)
  Duplicate class android.support.v4.media.MediaMetadataCompat$BitmapKey found in modules classes.jar (:androidx.media.media-1.0.0:) and classes.jar (:support-media-compat-25.1.0:)
  Duplicate class android.support.v4.media.MediaMetadataCompat$Builder found in modules classes.jar (:androidx.media.media-1.0.0:) and classes.jar (:support-media-compat-25.1.0:)
  Duplicate class android.support.v4.media.MediaMetadataCompat$LongKey found in modules classes.jar (:androidx.media.media-1.0.0:) and classes.jar (:support-media-compat-25.1.0:)
  Duplicate class android.support.v4.media.MediaMetadataCompat$RatingKey found in modules classes.jar (:androidx.media.media-1.0.0:) and classes.jar (:support-media-compat-25.1.0:)
  Duplicate class android.support.v4.media.MediaMetadataCompat$TextKey found in modules classes.jar (:androidx.media.media-1.0.0:) and classes.jar (:support-media-compat-25.1.0:)
  Duplicate class android.support.v4.media.MediaMetadataCompatApi21 found in modules classes.jar (:androidx.media.media-1.0.0:) and classes.jar (:support-media-compat-25.1.0:)
  Duplicate class android.support.v4.media.MediaMetadataCompatApi21$Builder found in modules classes.jar (:androidx.media.media-1.0.0:) and classes.jar (:support-media-compat-25.1.0:)
  Duplicate class android.support.v4.media.ParceledListSliceAdapterApi21 found in modules classes.jar (:androidx.media.media-1.0.0:) and classes.jar (:support-media-compat-25.1.0:)
  Duplicate class android.support.v4.media.RatingCompat found in modules classes.jar (:androidx.media.media-1.0.0:) and classes.jar (:support-media-compat-25.1.0:)
  Duplicate class android.support.v4.media.RatingCompat$1 found in modules classes.jar (:androidx.media.media-1.0.0:) and classes.jar (:support-media-compat-25.1.0:)
  Duplicate class android.support.v4.media.RatingCompat$StarStyle found in modules classes.jar (:androidx.media.media-1.0.0:) and classes.jar (:support-media-compat-25.1.0:)
  Duplicate class android.support.v4.media.RatingCompat$Style found in modules classes.jar (:androidx.media.media-1.0.0:) and classes.jar (:support-media-compat-25.1.0:)
  Duplicate class android.support.v4.media.session.IMediaControllerCallback found in modules classes.jar (:androidx.media.media-1.0.0:) and classes.jar (:support-media-compat-25.1.0:)
  Duplicate class android.support.v4.media.session.IMediaControllerCallback$Stub found in modules classes.jar (:androidx.media.media-1.0.0:) and classes.jar (:support-media-compat-25.1.0:)
  Duplicate class android.support.v4.media.session.IMediaControllerCallback$Stub$Proxy found in modules classes.jar (:androidx.media.media-1.0.0:) and classes.jar (:support-media-compat-25.1.0:)
  Duplicate class android.support.v4.media.session.IMediaSession found in modules classes.jar (:androidx.media.media-1.0.0:) and classes.jar (:support-media-compat-25.1.0:)
  Duplicate class android.support.v4.media.session.IMediaSession$Stub found in modules classes.jar (:androidx.media.media-1.0.0:) and classes.jar (:support-media-compat-25.1.0:)
  Duplicate class android.support.v4.media.session.IMediaSession$Stub$Proxy found in modules classes.jar (:androidx.media.media-1.0.0:) and classes.jar (:support-media-compat-25.1.0:)
  Duplicate class android.support.v4.media.session.MediaControllerCompat found in modules classes.jar (:androidx.media.media-1.0.0:) and classes.jar (:support-media-compat-25.1.0:)
  Duplicate class android.support.v4.media.session.MediaControllerCompat$Callback found in modules classes.jar (:androidx.media.media-1.0.0:) and classes.jar (:support-media-compat-25.1.0:)
  Duplicate class android.support.v4.media.session.MediaControllerCompat$Callback$MessageHandler found in modules classes.jar (:androidx.media.media-1.0.0:) and classes.jar (:support-media-compat-25.1.0:)
  Duplicate class android.support.v4.media.session.MediaControllerCompat$Callback$StubApi21 found in modules classes.jar (:androidx.media.media-1.0.0:) and classes.jar (:support-media-compat-25.1.0:)
  Duplicate class android.support.v4.media.session.MediaControllerCompat$Callback$StubCompat found in modules classes.jar (:androidx.media.media-1.0.0:) and classes.jar (:support-media-compat-25.1.0:)
  Duplicate class android.support.v4.media.session.MediaControllerCompat$MediaControllerExtraData found in modules classes.jar (:androidx.media.media-1.0.0:) and classes.jar (:support-media-compat-25.1.0:)
  Duplicate class android.support.v4.media.session.MediaControllerCompat$MediaControllerImpl found in modules classes.jar (:androidx.media.media-1.0.0:) and classes.jar (:support-media-compat-25.1.0:)
  Duplicate class android.support.v4.media.session.MediaControllerCompat$MediaControllerImplApi21 found in modules classes.jar (:androidx.media.media-1.0.0:) and classes.jar (:support-media-compat-25.1.0:)
  Duplicate class android.support.v4.media.session.MediaControllerCompat$MediaControllerImplApi21$ExtraCallback found in modules classes.jar (:androidx.media.media-1.0.0:) and classes.jar (:support-media-compat-25.1.0:)
  Duplicate class android.support.v4.media.session.MediaControllerCompat$MediaControllerImplApi23 found in modules classes.jar (:androidx.media.media-1.0.0:) and classes.jar (:support-media-compat-25.1.0:)
  Duplicate class android.support.v4.media.session.MediaControllerCompat$MediaControllerImplApi24 found in modules classes.jar (:androidx.media.media-1.0.0:) and classes.jar (:support-media-compat-25.1.0:)
  Duplicate class android.support.v4.media.session.MediaControllerCompat$MediaControllerImplBase found in modules classes.jar (:androidx.media.media-1.0.0:) and classes.jar (:support-media-compat-25.1.0:)
  Duplicate class android.support.v4.media.session.MediaControllerCompat$PlaybackInfo found in modules classes.jar (:androidx.media.media-1.0.0:) and classes.jar (:support-media-compat-25.1.0:)
  Duplicate class android.support.v4.media.session.MediaControllerCompat$TransportControls found in modules classes.jar (:androidx.media.media-1.0.0:) and classes.jar (:support-media-compat-25.1.0:)
  Duplicate class android.support.v4.media.session.MediaControllerCompat$TransportControlsApi21 found in modules classes.jar (:androidx.media.media-1.0.0:) and classes.jar (:support-media-compat-25.1.0:)
  Duplicate class android.support.v4.media.session.MediaControllerCompat$TransportControlsApi23 f<message truncated>`
